### PR TITLE
feature/NUI-2883 Wizard step header separation for wizard 1

### PR DESCRIPTION
**Pull request by dmytrognatyshyn**

### CODE REVIEW

File: packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                  required="true"
+                  [value]="false" i18n>Check box with help text
+    </nui-checkbox>
+</ng-template>
+
+<nui-wizard id="nui-demo-wizard"
+            finishText="Finish"
+            i18n-finishText>
+    <nui-wizard-step id="nui-demo-wizard-step" i18n-title title="Heading for first step content" shortTitle="First step" [stepTemplate]="step1"></nui-wizard-step>
+    <nui-wizard-step i18n-title title="Heading for final step content" shortTitle="Final step" [stepTemplate]="step2"></nui-wizard-step>
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html:26]:
```suggestion
    <nui-wizard-step i18n-title title="Heading for final step content" i18n-shortTitle shortTitle="Final step" [stepTemplate]="step2"></nui-wizard-step>
```

File: packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<ng-template #step1>
+    <nui-textbox
+        i18n-caption
+        caption="First"
+        name="hint"
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html:5]: Maybe this instead to be consistent with the checkbox name below:

```suggestion
        name="nui-demo-textbox"
```

File: packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        [value]="hint"
+        customBoxWidth="150px"
+        (textChange)="onOptionChange($event)">
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html:8]: Maybe this:

```suggestion
        (textChange)="onTextboxChange($event)">
```

File: packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Component } from "@angular/core";
+
+@Component({
+    selector: "nui-wizard-with-separate-step-headings-example",
+    templateUrl: "./wizard-with-separate-step-headings-example.component.html",
+})
+export class WizardWithSeparateStepHeadingsExampleComponent {
+    public hint = $localize `example-hint`;
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.ts:8]: Maybe this. Also, there's no need to try to localize a user input value.

```suggestion
    public textboxValue = "Example Textbox Value";
```

File: packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+                  hint="This is some help text"
+                  required="true"
+                  [value]="false" i18n>Check box with help text
+    </nui-checkbox>
+</ng-template>
+
+<nui-wizard id="nui-demo-wizard"
+            finishText="Finish"
+            i18n-finishText>
+    <nui-wizard-step id="nui-demo-wizard-step" i18n-title title="Heading for first step content" shortTitle="First step" [stepTemplate]="step1"></nui-wizard-step>
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-with-separate-step-headings/wizard-with-separate-step-headings-example.component.html:25]: Missing i18n:

```suggestion
    <nui-wizard-step id="nui-demo-wizard-step" i18n-title title="Heading for first step content" i18n-shortTitle shortTitle="First step" [stepTemplate]="step1"></nui-wizard-step>
```

File: packages/bits/demo/src/components/demo/wizard/wizard-docs/wizard-docs.example.component.html [hunk 6]
```diff
@@ -19,6 +19,14 @@ <h2>Basic Usage</h2>
     <nui-wizard-simple-example></nui-wizard-simple-example>
 </nui-example-wrapper>
 
+<h2>Wizard with separate step headings</h2>
+<p>
+    You can set separate headings for the step name and for the step content
```
Comment (Blakesalot) [inline packages/bits/demo/src/components/demo/wizard/wizard-docs/wizard-docs.example.component.html:24]: Maybe this:

```suggestion
    You can set separate values for the step name to show in the wizard header and the step content. 
```

**Review by ivankozlovskyi — APPROVED** [review]

**Review by DmytroPutrin — APPROVED** [review]

**Review by dmytrofirsov — APPROVED** [review]

**Pull request merged**